config/reports.json

```diff
@@ -1317,38 +1317,22 @@
 	},
 	"project-fugu": {
 		"name": "Capabilities",
-		"summary": "The [Capabilities Project](https://web.dev/fugu-status/) (aka. Project Fugu&nbsp;🐡) is a cross-company effort at Google to make it possible for web apps to do anything native apps can, by exposing the capabilities of native platforms to the web platform, while maintaining user security, privacy, trust, and other core tenets of the web.",
+		"summary": "The [Capabilities Project](https://web.dev/fugu-status/) (aka. Project Fugu&nbsp;🐡) is a cross-company effort at Google to make it possible for web apps to do anything native apps can, by exposing the capabilities of the operating system platform to the web platform, while maintaining user security, privacy, trust, and other core tenets of the web.",
 		"metrics": [
 			"asyncClipboardRead",
-			"asyncClipboardWrite",
 			"badgeSet",
 			"badgeClear",
-			"contactPicker",
 			"getInstalledRelatedApps",
 			"periodicBackgroundSyncRegister",
 			"periodicBackgroundSync",
-			"shapeDetectionBarcode",
 			"storageEstimate",
 			"storagePersist",
-			"webOTP",
-			"webUSB",
-			"webBluetooth",
-			"webMIDI",
 			"screenWakeLock",
-			"textFragment",
-			"webShareContainingFiles",
-			"webShareNoFiles",
 			"notificationTriggers",
-			"webNFCRead",
-			"webNFCWrite",
-			"webSerial",
-			"nativeFileSystem",
 			"contentIndex",
 			"quicTransport",
 			"idleDetection",
-			"webSocketStream",
-			"shapeDetectionFace",
-			"shapeDetectionText"
```

Comment:
If these features can never be detected in HA, should we remove their metric definitions above and corresponding SQL files in https://github.com/HTTPArchive/bigquery/pull/88?

Reply:
If the collection methodology changes, could this table contain data in the future?

Reply:
Theoretically, but I don't foresee HA simulating user gestures anytime soon.

Reply:
Let me axe them then. They're easy to re-establish should we need them in the future.